src/index.tsx

```diff
@@ -7,14 +7,15 @@ import reportWebVitals from './reportWebVitals';
 import { Provider } from 'react-redux';
 import { store } from './redux/configure-store';
 import { ChakraProvider } from '@chakra-ui/react';
+import { amethystTheme } from './themes/amethyst/index';
```

Comment:
super nit, drop the "theme" part of it, since we can kind of assume that any types coming out the themes directory are going to be themes.

Reply:
good point, done